tests/keras/legacy/interface_test.py

```diff
@@ -27,6 +27,19 @@ def test_dense_legacy_interface():
                                    bias_constraint='max_norm', name='d')
     assert json.dumps(old_layer.get_config()) == json.dumps(new_layer.get_config())
 
+@keras_test
+def test_embedding_legacy_interface():
+    old_layer = keras.layers.Embedding(input_dim=4, output_dim=2, name='d',
+                                   init='normal',
```

Comment:
You should use a PEP8 linter to avoid style issues. https://pypi.python.org/pypi/pep8